editor/icons/Breakpoint.svg

```diff
@@ -0,0 +1,16 @@
+<svg
+	enable-background="new 0 0 16 16"
+	height="16"
+	viewBox="0 0 16 16"
+	width="16"
+	xmlns="http://www.w3.org/2000/svg"
+>
+	<path
+		d="m15 8c0 3.866-3.134 7-7 7s-7-3.134-7-7 3.134-7 7-7 7 3.134 7 7"
+		fill="#e1e1e1"
+	/>
+	<path
+		d="m12 8c0 2.209-1.791 4-4 4s-4-1.791-4-4 1.791-4 4-4 4 1.791 4 4"
+		fill="#f6f6f6"
+	/>
+</svg>
```

Comment:
`svgo` can cut it to a half size, 0.34 KiB → 0.186 KiB

```suggestion
<svg height="16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="M15 8A7 7 0 1 1 1 8a7 7 0 0 1 14 0" fill="#e1e1e1"/><path d="M12 8a4 4 0 1 1-8 0 4 4 0 0 1 8 0" fill="#f6f6f6"/></svg>

```

Reply:
Oh sorry, I auto-formatted it in VS code to easily view it but forgot to compress it back, thanks.